_maps/map_files/MetaStation/MetaStation.dmm

```diff
@@ -80718,6 +80718,20 @@
 /obj/machinery/rnd/protolathe/department/service,
 /turf/closed/wall,
 /area/hydroponics)
+"EDx" = (
+/turf/open/space,
+/turf/closed/wall/mineral/plastitanium{
+	dir = 8;
+	icon_state = "diagonalWall3"
```

Comment:
What's with varedited icon states here ? These should smooth?

Reply:
Those are walls surrounding the escape pods; nothing I actually changed, just put an area over them.

Reply:
Alright